sfputil/main.py

```diff
@@ -917,28 +917,25 @@ def fetch_error_status_from_platform_api(port):
 
     output_list = output.split('\n')
     for output_str in output_list:
-        # The output of all SFP error status are a list consisting of element with convention of '<sfp no>:<error status>'
+        # The output of all SFP error status are a list consisting of <error status> sorted physical port indices
         # Besides, there can be some logs captured during the platform API executing
         # So, first of all, we need to skip all the logs until find the output list of SFP error status
         if output_str[0] == '[' and output_str[-1] == ']':
             output_list = ast.literal_eval(output_str)
             break
 
-    output_dict = {}
-    for output in output_list:
-        sfp_index, error_status = output.split(':')
-        output_dict[int(sfp_index)] = error_status
+    physical_port_set = set()
```

Comment:
what's the motivation to introduce a set `physical_port_set`? Is it possible to use list directly?

Reply:
@stephenxs this is because to avoid duplicate indices for platform that may have breakout ports so there will be multiple logical ports having same physical indices.